cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -35,7 +35,11 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 
 	flags.Uint64("update-parallelism", 0, "task update parallelism (0 = all at once)")
 	flags.String("update-delay", "0s", "delay between task updates (0s = none)")
-	flags.String("update-on-failure", "pause", "action on failure during update (pause|continue)")
+	flags.String("update-on-failure", "pause", "action on failure during update (pause|continue|rollback)")
+
+	flags.Uint64("rollback-parallelism", 0, "task update parallelism during rollback (0 = all at once)")
```

Comment:
This default doesn't sound right. It'd interrupt service. Same with `update-parallelism`. How about changing default to 2 halves?